### ISSUE TITLE
missing index.d.ts file in latest NPM package

### ISSUE THREAD

**Issue by jvert** [googlemaps/react-wrapper#152]
dist/index.d.ts file is missing in npm packages after 1.1.1

#### Steps to reproduce
Just look at the package contents - 1.1.1 contains dist/index.d.ts, 1.1.2 does not.

OR 

```
npx create-react-app googlemap --template typescript
cd googlemap
npm install @googlemaps/react-wrapper
```  
add this line to src/App.tsx
`import { Wrapper } from "@googlemaps/react-wrapper";`

Try to build: `npm run build`

#### Error message
```
Failed to compile.

C:/Users/john/source/googlemap/src/App.tsx
TypeScript error in C:/Users/john/source/googlemap/src/App.tsx(4,25):
Could not find a declaration file for module '@googlemaps/react-wrapper'. 'C:/Users/john/source/googlemap/node_modules/@googlemaps/react-wrapper/dist/index.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/googlemaps__react-wrapper` if it exists or add a new declaration (.d.ts) file containing `declare module '@googlemaps/react-wrapper';`  TS7016

    2 | import logo from "./logo.svg";
    3 | import "./App.css";
  > 4 | import { Wrapper } from "@googlemaps/react-wrapper";
      |                         ^
    5 |
    6 | function App() {
    7 |   return (
```

**Comment by jpoehnelt** [comment]
Now I feel a bit embarrassed... Fix incoming.

**Comment by jpoehnelt** [comment]
@jvert let me know if there are any other issues!

**Comment by 0x70b1a5** [comment]
Hi there. I think this issue has reared its ugly head again. :( Same steps to reproduce.

**Comment by jpoehnelt** [comment]
Fix incoming. Seems to be from moving to the official rollup plugin which doesn't automatically pull in the correct config. https://github.com/rollup/plugins/issues/394

**Comment by 0x70b1a5** [comment]
Not sure if this is related, but I think the same issue is happening here https://github.com/googlemaps/js-typescript-guards/ - do you want me to open an issue there as well?

**Comment by jpoehnelt** [comment]
Only if you don't see a fix in the next few hours!

**Comment by jpoehnelt** [comment]
fixed in #267 

wrote a github action to check this that I am still rolling out: https://github.com/jpoehnelt/verify-npm-files-action

**Comment by jpoehnelt** [comment]
version 1.1.14 should have the typings added back.
